Tools/TestWebKitAPI/Tests/WebCore/CARingBufferTest.cpp

```diff
@@ -105,11 +105,11 @@ TEST_F(CARingBufferTest, Basics)
     auto fetchBounds = ringBuffer().getFetchTimeBounds();
     EXPECT_EQ(fetchBounds, makeBounds(0, 0));
 
-    float sourceBuffer[capacity];
+    std::array<float, capacity> sourceBuffer;
     for (size_t i = 0; i < capacity; i++)
         sourceBuffer[i] = i + 0.5;
 
-    setListDataBuffer(reinterpret_cast<uint8_t*>(sourceBuffer), capacity);
+    setListDataBuffer(reinterpret_cast<uint8_t*>(sourceBuffer.data()), capacity);
```

Comment:
We should move this `reinterpret_cast` and call to `data()` inside the `setListDataBuffer` function. But I guess there's a limit to how much I care about that in test code.